### ISSUE TITLE
[torch_xla] Performance of Llama 3.1 405B matches Huggingface fork on 1 pod

### ISSUE THREAD

**Issue by tengyifei** [AI-Hypercomputer/torchprime#73]
It's easier to bring up the model on 1 pod. We should do that and verify that the perf is as good as the recipe in https://github.com/AI-Hypercomputer/tpu-recipes/tree/main/training/trillium/Llama3-405B-PyTorch/GCE.

- [x] Merge https://github.com/pytorch/xla/issues/8381
- [ ] Do not use `HybridMesh` on one pod, until we can roll to the latest torch_xla nightly build (blocked on https://github.com/pytorch/xla/issues/8683)
- [ ] Add Llama 3.1 405B config
- [ ] Test that the loss is not NaN

**Comment by tengyifei** [comment]
I believe for this we just need https://github.com/AI-Hypercomputer/torchprime/issues/22.

**Comment by tengyifei** [comment]
Fixed by https://github.com/AI-Hypercomputer/torchprime/pull/108
